### ISSUE TITLE
Does face need aligned in predict.py?

### ISSUE THREAD

**Issue by GuitarZhang** [tornadomeet/mxnet-face#33]
I found that in predict.py only the croped face are put into net  to predict the attributes,  the face is not  aligned, is it right?

**Comment by tornadomeet** [comment]
yes, you're right. because no align in training.  you man consider using align during training.
